### PR TITLE
fix: update styles on color page

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes https://github.com/patternfly/patternfly-org/issues/4067
Closes https://github.com/patternfly/patternfly-org/issues/4073

### CODE REVIEW

File: packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/colors.md [hunk 1]
```diff
@@ -174,4 +188,43 @@ Status and state colors are indicators that communicate data and actions to user
...
+To check the contrast between background and text colors, use a <a href="https://color.a11y.com/?wc3" target="_blank" className="pf-m-link">WCAG AA-compliance tool.</a>
+
+<Divider/>
+
+## PatternFly palettes
+Our color palettes are created as a system, designed to reinforce content and workflows. Our palettes are carefully chosen to support effective communication and adapt to the needs of your UI.
+
+Note that bright colors are used sparingly and are reserved for specific interactions, such as communicating status.
+
+Our color palettes are organized into "families" that contain different shades of the same hue. In the following palettes, you can expand each value color or shadow in a family to see more details.
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/colors.md:200]:
```suggestion
## Color families 
Our color palettes are organized into "families" that contain different shades of the same hue. In the following families, you can expand each color to see related tokens.
```

Did we decide to keep the color families in addition to linking out to the Red Hat palettes? I think I removed the whole section in my pr, so would update the text to something like this if we are going to keep them

**Comment by edonehoo** [comment]
A few things I noticed: 

1. In dark mode, the text on the color families doesn't update and can't be read in some cases: 
image

2. Also in the families, there are sometimes no tokens listed (which maybe there are just no tokens related to the color? but it looks like there should be some listed): 
image

3. Related to ^ Should we change the label from "CSS variables" to "Tokens" in the expanded family sections?

4. This is small, but the primary background color swatch has a background, even before you hover:
image

File: packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/colors.md [hunk 2]
```diff
@@ -174,4 +188,43 @@ Status and state colors are indicators that communicate data and actions to user
 
 [Our goal](/accessibility/patternflys-accessibility/) is for PatternFly to meet [level AA requirements in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/). To achieve level AA accessibility, your UI contrast ratios must be at or above 4.5:1 for normal text, 3:1 for large text, and 3:1 for graphics and other UI components. Additionally, on hover, link text color should have ample contrast from both the background color and the default state link color.
 
-To check the contrast between background and text colors, use a <a href="https://color.a11y.com/?wc3" target="_blank" className="pf-m-link">WCAG AA-compliance tool.</a>
\ No newline at end of file
+To check the contrast between background and text colors, use a <a href="https://color.a11y.com/?wc3" target="_blank" className="pf-m-link">WCAG AA-compliance tool.</a>
+
+<Divider/>
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/colors.md:193]:
```suggestion
```

**Comment by nicolethoen** [comment]
> 4\. This is small, but the primary background color swatch has a background, even before you hover:

That's because without a background color, the color swatch is invisible! 😆 I'm open to other color suggestions!

> Also in the families, there are sometimes no tokens listed (which maybe there are just no tokens related to the color? but it looks like there should be some listed):

I can make a 'no semantic tokens message if there are none?

**Comment by edonehoo** [comment]
> That's because without a background color, the color swatch is invisible!
Oh, duh! 😆 I can't think of a different approach that's ~the golden ticket, so I think we can just stick with this one, just to get the info updated? And then maybe revisit with a designer to see if we want to change it

**Review by edonehoo — APPROVED** [review]
I like it 🗣️

File: packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/colors.md [hunk 3]
```diff
@@ -174,4 +188,37 @@ Status and state colors are indicators that communicate data and actions to user
...
+    <ColorFamily title="Gray family" family="gray" />
+  </GridItem>
+  <GridItem xl2={4} xl={6} sm={12}>
+    <ColorFamily title="Blue family" family="blue" />
+  </GridItem>
+  <GridItem xl2={4} xl={6} sm={12}>
+    <ColorFamily title="Green family" family="green" />
+  </GridItem>
+  <GridItem xl2={4} xl={6} sm={12}>
+    <ColorFamily title="Cyan family" family="teal" />
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/colors.md:207]:
```suggestion
    <ColorFamily title="Teal family" family="teal" />
```
wait, just noticed this. I think we call this teal now

**Review by edonehoo — CHANGES_REQUESTED** [review]

**Review by edonehoo — APPROVED** [review]

**Pull request merged**